lib/swaps/Swaps.ts

```diff
@@ -426,7 +433,7 @@ class Swaps extends EventEmitter {
     }
 
     const responseBody: packets.SwapAcceptedPacketBody = {
-      makerCltvDelta: deal.makerCltvDelta || 0,
+      makerCltvDelta: deal.makerCltvDelta || 1,
```

Comment:
The reason for this change is that the `SwapAcceptedPacket` will not validate otherwise:
https://github.com/ExchangeUnion/xud/blob/master/lib/p2p/packets/types/SwapAcceptedPacket.ts#L36